src/modules/launcher/PowerLauncher/ViewModel/ContextMenuItemViewModel.cs

```diff
@@ -15,11 +15,50 @@ public class ContextMenuItemViewModel : BaseModel
 
         public string PluginName { get; set; }
```

Comment:
Will this need to be a tweaked get / set?

Reply:
The public `set` seems wrong, it should be set in the constructor
https://github.com/microsoft/PowerToys/issues/9172

Reply:
No, it does not need to be tweaked as it is neither: 
- Code subscribed property
- Property that is used in xaml file